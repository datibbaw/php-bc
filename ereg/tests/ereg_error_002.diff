004+ int(1)
005- Warning: ereg(): REG_EMPTY in %s on line %d
006+ Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81
007+ 
008+ Warning: preg_match(): Compilation failed: missing ) at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
010- Warning: ereg(): REG_EPAREN in %s on line %d
013+ Warning: preg_match(): Compilation failed: missing terminating ] for character class at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
015- Warning: ereg(): REG_EBRACK in %s on line %d
018+ Warning: preg_match(): Compilation failed: missing ) at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
020- Warning: ereg(): REG_EPAREN in %s on line %d
023+ Warning: preg_match(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
025- Warning: ereg(): REG_BADRPT in %s on line %d
028+ Warning: preg_match(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
030- Warning: ereg(): REG_BADRPT in %s on line %d
033+ Warning: preg_match(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
035- Warning: ereg(): REG_BADRPT in %s on line %d
038+ Warning: preg_match(): Compilation failed: nothing to repeat at offset 1 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
040- Warning: ereg(): REG_BADRPT in %s on line %d
041- bool(false)
042+ bool(false)
044+ Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81
045+ int(1)
046+ 
047+ Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81
048+ int(1)
045- Warning: ereg(): REG_BADBR in %s on line %d
050- Warning: ereg(): REG_EMPTY in %s on line %d
052+ Warning: preg_match(): Compilation failed: numbers out of order in {} quantifier at offset 5 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
055- Warning: ereg(): REG_EMPTY in %s on line %d
056+ bool(false)
057- 
058- Deprecated: Function ereg() is deprecated in %s on line %d
058+ Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81
059+ 
060+ Warning: preg_match(): No ending delimiter '/' found in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
065+ Warning: preg_match(): Compilation failed: range out of order in character class at offset 4 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
067+ string(8) "original"
068+ Done
059- 
060- Warning: ereg(): REG_BADBR in %s on line %d
061- bool(false)
062- 
063- Deprecated: Function ereg() is deprecated in %s on line %d
064- 
065- Warning: ereg(): REG_ERANGE in %s on line %d
066- bool(false)
067- 
068- Deprecated: Function ereg() is deprecated in %s on line %d
069- 
070- Warning: ereg(): REG_EESCAPE in %s on line %d
071- bool(false)
072- 
073- Deprecated: Function ereg() is deprecated in %s on line %d
074- 
075- Warning: ereg(): REG_ERANGE in %s on line %d
076- bool(false)
077- string(8) "original"
078- Done