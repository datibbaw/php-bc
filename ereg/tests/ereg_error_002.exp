*** Testing ereg() : error conditions ***

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_EMPTY in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_EPAREN in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_EBRACK in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_EPAREN in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_BADRPT in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_BADRPT in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_BADRPT in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_BADRPT in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_BADBR in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_EMPTY in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_EMPTY in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_BADBR in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_ERANGE in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_EESCAPE in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_ERANGE in %s on line %d
bool(false)
string(8) "original"
Done