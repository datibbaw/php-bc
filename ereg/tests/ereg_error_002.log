
---- EXPECTED OUTPUT
*** Testing ereg() : error conditions ***

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_EMPTY in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_EPAREN in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_EBRACK in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_EPAREN in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_BADRPT in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_BADRPT in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_BADRPT in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_BADRPT in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_BADBR in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_EMPTY in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_EMPTY in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_BADBR in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_ERANGE in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_EESCAPE in %s on line %d
bool(false)

Deprecated: Function ereg() is deprecated in %s on line %d

Warning: ereg(): REG_ERANGE in %s on line %d
bool(false)
string(8) "original"
Done
---- ACTUAL OUTPUT
*** Testing ereg() : error conditions ***

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81
int(1)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81

Warning: preg_match(): Compilation failed: missing ) at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81

Warning: preg_match(): Compilation failed: missing terminating ] for character class at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81

Warning: preg_match(): Compilation failed: missing ) at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81

Warning: preg_match(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81

Warning: preg_match(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81

Warning: preg_match(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81

Warning: preg_match(): Compilation failed: nothing to repeat at offset 1 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81
int(1)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81
int(1)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81

Warning: preg_match(): Compilation failed: numbers out of order in {} quantifier at offset 5 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81

Warning: preg_match(): No ending delimiter '/' found in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81

Warning: preg_match(): Compilation failed: range out of order in character class at offset 4 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
bool(false)
string(8) "original"
Done
---- FAILED
