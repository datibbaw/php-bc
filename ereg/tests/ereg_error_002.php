<?php
/* Prototype  : proto int ereg(string pattern, string string [, array registers])
 * Description: Regular expression match 
 * Source code: ext/standard/reg.c
 * Alias to functions: 
 */

/*
 * Test bad regular expressions
 */

echo "*** Testing ereg() : error conditions ***\n";

$regs = 'original';

var_dump(ereg("", "hello"));
var_dump(ereg("c(d", "hello"));
var_dump(ereg("a[b", "hello"));
var_dump(ereg("c(d", "hello"));
var_dump(ereg("*", "hello"));
var_dump(ereg("+", "hello"));
var_dump(ereg("?", "hello"));
var_dump(ereg("(+?*)", "hello", $regs));
var_dump(ereg("h{256}", "hello"));
var_dump(ereg("h|", "hello"));
var_dump(ereg("h{0}", "hello"));
var_dump(ereg("h{2,1}", "hello"));
var_dump(ereg('[a-c-e]', 'd'));
var_dump(ereg('\\', 'x'));
var_dump(ereg('([9-0])', '1', $regs));

//ensure $regs unchanged
var_dump($regs);

echo "Done";
?>
