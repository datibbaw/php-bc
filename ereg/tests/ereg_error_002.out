*** Testing ereg() : error conditions ***

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81
int(1)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81

Warning: preg_match(): Compilation failed: missing ) at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81

Warning: preg_match(): Compilation failed: missing terminating ] for character class at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81

Warning: preg_match(): Compilation failed: missing ) at offset 3 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81

Warning: preg_match(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81

Warning: preg_match(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81

Warning: preg_match(): Compilation failed: nothing to repeat at offset 0 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81

Warning: preg_match(): Compilation failed: nothing to repeat at offset 1 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81
int(1)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81
int(1)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81

Warning: preg_match(): Compilation failed: numbers out of order in {} quantifier at offset 5 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81

Warning: preg_match(): No ending delimiter '/' found in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
bool(false)

Deprecated: Function ereg() is deprecated in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 81

Warning: preg_match(): Compilation failed: range out of order in character class at offset 4 in /Users/tjerk/Documents/php-bc/ereg/ereg.php on line 83
bool(false)
string(8) "original"
Done